client/__tests__/AddReviewModal.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, screen, cleanup, fireEvent } from '@testing-library/react';
+
+import ProductDetailPage from '../src/components/ProductDetailPage.jsx';
+
+beforeEach(() => {
+  var ProductDetailPageMock = new ProductDetailPage;
+  render(<ProductDetailPage />);
+  ProductDetailPageMock.hideModal();
+});
+
+afterEach(() => {
+  cleanup();
+});
+
+test('renders react-modal', () => {
```

Comment:
I don't think we need this test here, since we have the modal tests that do this: 
<img width="380" alt="Screen Shot 2021-06-24 at 7 39 35 PM" src="https://user-images.githubusercontent.com/65684361/123361458-f3747780-d523-11eb-9359-35f876a3cb39.png">

Reply:
Want to keep things compact so if there is a change in the modal workings, we don't have to change it three hundred places (obvs hyperbole, but still)

Reply:
Absolutely agree. I considered both options, but didn't know if something changes in the modal tests, would I still want to test this aspect of the modal in my component.